simplebnf-doc.tex

```diff
@@ -1,82 +1,81 @@
 \documentclass[a4paper]{article}
 
-\usepackage{fontspec}
-\setmainfont{Linux Libertine}
-\setsansfont{Libertinus Sans}
-\setmonofont{Inconsolata}
-
+\usepackage{simplebnf}
 \usepackage{hyperref}
 
 \usepackage{tcolorbox}
 \tcbuselibrary{listings,breakable}
 \tcbset{listing engine=listings,colframe=black,colback=white,size=small}
+
 \NewDocumentEnvironment {exampleside} {}
-  { \tcblisting{listing side text,righthand width=.4\textwidth} }
+  { \tcblisting{listing side text,righthand width=.5\textwidth} }
   { \endtcblisting }
 
+\NewDocumentCommand \cmd { m } {\texttt{\textbackslash#1}}
+
 \NewDocumentEnvironment { presentcommand } { b }
   {%
     \vspace*{0.5\baselineskip}\noindent\fbox{%
-      \begin{minipage}{\dimexpr\textwidth-2\fboxsep-2\fboxrule}
-        #1
-      \end{minipage}}\vspace*{0.5\baselineskip}
+    \begin{minipage}{\dimexpr\textwidth-2\fboxsep-2\fboxrule}
+      #1
+    \end{minipage}}\vspace*{0.5\baselineskip}
   }
   { }
 
-\NewDocumentCommand \cmd { m } { \texttt{\textbackslash#1} }
 \NewDocumentCommand \env { m m }
   {
     \texttt{%
-      \textbackslash begin\{#1\} \textrm{#2} %
+      \textbackslash begin\{#1\} \textrm{#2}%
       \textbackslash end\{#1\}%
     }%
   }
 
-\usepackage{simplebnf}
-
 \title{%
   \textsf{simplebnf} --- A simple package to format Backus-Naur form%
-  \footnote{This file describes v0.1.0.}}
+  \footnote{This file describes v0.2.0.}}
 \author{Jay Lee\footnote{E-mail: %
   \href{mailto:jaeho.lee@snu.ac.kr}{\texttt{jaeho.lee@snu.ac.kr}}}}
-\date{2019/12/23}
+\date{2020/09/01}
 
 \begin{document}
 \maketitle
 
+This package provides a simple way to typeset grammars written in BNF form.
```

Comment:
_form_ in _BNF form_ is redundant, as BNF already is an acronym for Backus-Naur Form. IMHO, the first occurrence of the term BNF should be preceded by the full name, e.g., _Backus-Naur form(BNF)_.

Reply:
I've amended the commit with all requested changes.

Reply:
Thanks a lot!